components/prombench/manifests/results/grafana_deployment.yaml

```diff
@@ -91,3 +91,22 @@ spec:
   selector:
     app: grafana
     component: core
+
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: ingress-grafana
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
+    # see https://kubernetes.github.io/ingress-nginx/examples/rewrite/#rewrite-target
```

Comment:
I meant like a short description what is does in our case to avoid having to jump to the nginx description

IIUC this is so that any path with grafana in the url is redirected to this ingress right?